features/draft_environment.feature

```diff
@@ -32,9 +32,3 @@ Feature: Draft environment
     Then I should see "Content design"
     And I should see "Search this manual"
     And the page should contain the draft watermark
-
-  @draft
-  Scenario: visiting a page served by contacts-frontend
```

Comment:
Presumably this is a test to check that contacts-frontend is running, rather than a test that contacts are successfully rendering. If it's the latter then the scenario could be renamed.